GDJS/Runtime/layer.ts

```diff
@@ -463,10 +463,8 @@ namespace gdjs {
      * Return the time elapsed since the last frame,
      * in milliseconds, for objects on the layer.
      */
-    getElapsedTime(): float {
-      return (
-        this._runtimeScene.getTimeManager().getElapsedTime() * this._timeScale
-      );
+    getElapsedTime(runtimeScene: RuntimeScene): float {
+      return runtimeScene.getTimeManager().getElapsedTime() * this._timeScale;
```

Comment:
Sorry for the u turn, I think we can actually do the reverse decision:
- keep getElapsedTime without any parameter in the layer, and remove the parameter from gdjs.RuntimeObject too.
- the reason is that both the layer and the runtimeobject have a reference to the runtimescene, and there is little to no chance that in the future we want to drop this reference.
- it's no problem if some code is passing the runtimescene as an argument to the object getElapsedTime, we can just ignore it. You can make the argument optional.

This means updating the EffectsTarget and also the filters using getElapsedTime - but hopefully won't be too much work :)